.gitpod.yml

```diff
@@ -0,0 +1,25 @@
+tasks:
+  - init: |
+      mvn -am -pl war,bom -P quick-build install
+    command: |      
+      cd war
+      curl https://gist.githubusercontent.com/timja/fe73f0a0f98afe4d9356e7ce6752d95b/raw/98ad71080e91ed75824ec410ff905419e3788065/init.sh | bash
```

Comment:
Pulling down a shell script from a user's personal gist seems like a workaround to me. Can we not e.g. set a system property when invoking `mvn jetty:run` to customize this value? Seems like the ideal solution would be to make this customizable by a system property if it isn't already and then to pass that property to `mvn jetty:run`. This would present a (semi) supported interface for setting this value, whereas manually writing out an XML file is an awkward workaround and could become out of date in the future if the object model changes. If I am missing something and implementing a system property for this is either infeasible or a large amount of work, then I could possibly be convinced otherwise about the system property approach, but I doubt you will be able to convince me that putting a link to `https://gist.githubusercontent.com/timja` in Jenkins core is a good long-term solution. :smile:

Reply:
not going to debate this now, there's no interface / property I'm aware of and not inclined to do it at this time.
I've removed setting the property and users will just get prompted to save it once, (the same experience they currently get with `jetty:run` locally)